Doc/reference/compound_stmts.rst

```diff
@@ -378,8 +382,12 @@ exception group with an empty message string. ::
    ...
    ExceptionGroup('', (BlockingIOError()))
 
-An :keyword:`!except*` clause must have a matching type,
-and this type cannot be a subclass of :exc:`BaseExceptionGroup`.
+Unlike :keyword:`except` clauses, :keyword:`!except*` clauses must have a
+matching expression; ``except*:`` is not valid syntax. In addition to the
```

Comment:
(I wouldn't say it's invalid syntax when it generates a runtime error).

Reply:
It's a SyntaxError:
```
>>> try: x
... except *: pass
  File "<stdin>", line 2
    except *: pass
            ^
SyntaxError: expected one or more exception types
```